sites/public/src/ListingView.tsx

```diff
@@ -341,31 +328,7 @@ export const ListingView = (props: ListingProps) => {
           <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-gray-400 bg-white">
             <div className="hidden md:block">
               <ListingUpdated listing={listing} />
-              <ApplicationStatus content={appStatusContent} />
```

Comment:
Are there opportunities to remove the appStatusContent variable entirely? I'd imagine we're not using it anymore here.

Reply:
Yes. I've pulled all App and Date items out as well that were defined but never used.